onediff_diffusers_extensions/setup.py

```diff
@@ -12,7 +12,7 @@
     python_requires=">=3.7.0",
     install_requires=[
         "transformers>=4.27.1",
-        "diffusers>=0.24.0,<=0.26.2",
+        "diffusers>=0.24.0,<=0.25.1",
```

Comment:
relate issue: https://github.com/siliconflow/onediff/issues/636

revert to pass ci